tests/test_slogpy_console.py

```diff
@@ -0,0 +1,25 @@
+"""Import slogpy from slog"""
+from slogpy.slog import Slog as slog # noqa: N813 I001
```

Comment:
This project enforces style with ruff.  Generally speaking it is not okay to bypass the rules.  Please remove I001 and fix the code to match the style, ruff can probably do that for you automatically.

Reply:
ruff didn't do that for me. Instead, I installed and used "isort" to help me fix it.